config/application.yml.default

```diff
@@ -109,6 +109,7 @@ doc_auth_error_sharpness_threshold: 40
 doc_auth_max_attempts: 5
 doc_auth_max_capture_attempts_before_native_camera: 3
 doc_auth_max_submission_attempts_before_native_camera: 3
+doc_auth_mock_upload_enabled: false
```

Comment:
Is a new config necessary, or could we use `doc_auth_selfie_desktop_test_mode`? 
`doc_auth_mock_upload_enabled` feels difficult to understand when `doc_auth_vendor_default: 'mock'` is also an option

If possible, it'd be great to have the case where if `doc_auth_selfie_desktop_test_mode` is true, the desktop option is always presented and takes you to the upload screen.